src/components/Footer.js

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
 import { ReactComponent as Sign } from './sign.svg';
 import {
-  FooterStyled, LogoLink, FooterLink, FooterA,
+  FooterStyled, LogoLink, FooterLink,
 } from './Footer.style';
 
 function Footer() {
   return (
     <FooterStyled>
-      <FooterA href="https://ooloo.io" target="_blank">ooloo.io</FooterA>
+      <FooterLink as="a" href="https://ooloo.io" target="_blank">ooloo.io</FooterLink>
       <LogoLink to="/">
         <Sign />
       </LogoLink>
-      <FooterLink to="/terms">Terms & Privacy</FooterLink>
+      <FooterLink style={{ 'text-align': 'right' }} to="/terms">Terms & Privacy</FooterLink>
```

Comment:
What you can do here is to have a separate component in your style file like this

```
export const FooterTermsLink = styled(FooterLink)`
  text-align: right;
`
```

Would make it a bit more consistent. But I'll approve anyways and leave it up to you